source/Calamari.CloudAccounts/AzureOidcAccountExtensions.cs

```diff
@@ -22,12 +22,15 @@ public static Task<string> GetAuthorizationToken(this AzureOidcAccount account,
 
         public static async Task<string> GetAuthorizationToken(string tenantId, string applicationId, string token, string managementEndPoint, string activeDirectoryEndPoint, string aureEnvironment, CancellationToken cancellationToken)
         {
+            var authClientFactory = new AuthHttpClientFactory();
```

Comment:
Adding [recent fix](https://github.com/OctopusDeploy/Calamari/pull/1137/files) for proxies to OIDC auth calls